Added power drop faq. (#371)
filecoin-project/filecoin-docs: docs/mine/spacerace.md

  MaxWindowPoStGasFee = "50 FIL"
```

## Why did my miner power suddenly drop?

> The following are some of the common reasons that can cause your power to be drop:

Reviewer: ```suggestion
Here are some of the common reasons that your power can suddenly drop:
```

1. Your PoSt messages are blocked by other messages which leads to not submitting PoSt for a deadline/window. To fix this, you can follow this [guide](https://github.com/filecoin-project/lotus/blob/master/documentation/en/mining.md#separate-address-for-windowpost-messages) and setup an separate address just for windowPoSt messages.

2. Your sector data is corrupted and PoSt cannot be generated with bad underlying data. Unfortunately, we currently have no easy way to detect corrupted sectors, and the entire *runPost* fails even if only one sector is marked as faulty with the current proof APIs. However, the proofs team is working on getting our miners better APIs to mitigate this issue!

> 3. For smaller miners, when all sectors are put into a single window, missing just one deadline/window wipes out all the power for 24 hours. 

Reviewer: Can you include an explanation of when/why this resumes?

Author: Im actually not sure. @arajasek does this happen when  the miner does not have enough CPU to do all the proves?

Reviewer 2: @mishmosh Power will be recovered if a valid WindowPoSt is submitted 24 hours later, when the missed deadline reopens.

> 4. Chain/miner nodes crashed. Check `lotus sync status` in this case.

Reviewer: Add more info about what to look for & what the user can do about it. For example, check lotus sync status, if it says X then ___, if it says Y then ___.

> 3. For smaller miners, when all sectors are put into a single window, missing just one deadline/window wipes out all the power for 24 hours. Power will be recovered if a valid WindowPoSt is submitted 24 hours later, when the missed deadline reopens. Make sure that your miner is up and has enough CPU for PoSt.

Reviewer: ```suggestion
3. For smaller miners, when all sectors are put into a single window, missing just one deadline/window wipes out all the power for 24 hours. Power will automatically recover if a valid WindowPoSt is submitted 24 hours later, when the missed deadline reopens. Just make sure that your miner is running and has enough CPU for PoSt.
```

> 4. Chain/miner nodes crashed. Check `lotus sync wait` in this case, if your `Taget Height` is not equal to `Height` or status is not `Completed`, that means you are not fully synced.

Reviewer: ```suggestion
4. Chain/miner nodes crashed. Run `lotus sync wait` and check the output. If your `Target Height` is not equal to `Height` or status is not `Completed`, that means you are not fully synced.
```
